Gemfile

```diff
@@ -32,20 +32,20 @@ gem 'exception_notification'
 # Use Capistrano for deployment
 # gem 'capistrano-rails', group: :development
 
-gem 'coveralls', require: false
+# gem 'coveralls', require: false
```

Comment:
No longer needed?

Reply:
Yep, replaced by using simplecov directly. I've removed the comment now.